index.html

```diff
@@ -1210,25 +1218,25 @@ <h3>Types</h3>
         </table>
 
         <p>
-All <a>credentials</a>, <a>presentations</a>, and encapsulated objects MUST
-specify, or be associated with, additional more narrow <a>types</a> (like
-<code>UniversityDegreeCredential</code>, for example) so software systems can
-process this additional information.
+All <a>verifiable credentials</a>, <a>verifiable presentations</a>, and
+encapsulated objects MUST specify, or be associated with, additional more narrow
+<a>types</a> (like <code>UniversityDegreeCredential</code>, for example) so
+software systems can process this additional information.
         </p>
 
         <p>
 When processing encapsulated objects defined in this specification, (for
 example, objects associated with the <code>credentialSubject</code> object
 or deeply nested therein), software systems SHOULD use the <a>type</a>
 information specified in encapsulating objects higher in the hierarchy.
-Specifically, an encapsulating object, such as <a>credential</a>, SHOULD
-convey the associated object <a>types</a> so that <a>verifiers</a> can quickly
-determine the contents of an associated object based on the encapsulating
-object <a>type</a>.
+Specifically, an encapsulating object, such as a <a>verifiable credential</a>,
```

Comment:
This change is wrong... we meant "credential"

Reply:
Fixed.